functions/Export-SqlUser.ps1

```diff
@@ -153,7 +154,7 @@ https://dbatools.io/Export-SqlUser
 
         if ($databases.Count -eq 0)
         {
-            $databases = $sourceserver.Databases | Where-Object {$exclude -notcontains $_.Name -and $_.IsSystemObject -eq $false -and $_.IsAccessible -eq $true}
+            $databases = $sourceserver.Databases | Where-Object {$exclude -notcontains $_.Name -and $_.IsSystemObject -eq $IncludeSystemDatabases -and $_.IsAccessible -eq $true}
```

Comment:
Include means adding to existing one. Which means that if I say $IncludeSystemDatabases = $true I want to validate system databases along with user database.
The way the validation is done will show only system databases **or** user databases, never both.
imo, both should be the default.
What do you think?